drivers/frequency/adf4368/adf4368.c

```diff
@@ -0,0 +1,1953 @@
+			bleed_pol = dev->n_int < ADF4368_BLEED_N_INT_TH ?
+				    true : false;
+		}
+		/* 1.8GHz < RFout < 1.2GHz */
+		else if (dev->freq > 1200000000UL) {
+			t_bleed = 1400;
+			bleed_pol = dev->n_int < ADF4368_BLEED_N_INT_TH ?
+				    true : false;
+		}
+		/* dev->freq < 1.2 GHz */
+		else {
+			t_bleed = dev->n_int < ADF4368_BLEED_N_INT_TH ?
+				  2000 : 1400;
+			bleed_pol = dev->n_int < ADF4368_BLEED_N_INT_TH ?
+				    true : false;
+		}
+
+	}
+
+	/* Calculate Bleed Current */
+	bleed_i = t_bleed * (pfd_freq * adf4368_ci_ua[dev->cp_i]);
+
+	bleed_i = NO_OS_DIV_ROUND_UP(bleed_i, ADF4368_PS_TO_S);
+
+	coars_bleed = no_os_div64_u64_rem(bleed_i, ADF4368_COARSE_BLEED_CNST, &rem);
+
+	fine_bleed = rem * ADF4368_FINE_BLEED_CNST2;
+	fine_bleed = NO_OS_DIV_ROUND_UP(fine_bleed, ADF4368_FINE_BLEED_CNST1);
+
+	dev->bleed_word = (coars_bleed << 9) | fine_bleed;
+
+
+	/* Set Bleed Pol */
+	ret =  adf4368_spi_update_bits(dev, 0x1F,
+				       ADF4368_BLEED_POL_MSK,
+				       no_os_field_prep(ADF4368_BLEED_POL_MSK, bleed_pol));
+	if (ret)
+		return ret;
+
+
+	val = dev->bleed_word & ADF4368_FINE_BLEED_LSB_MSK;
+	ret = adf4368_spi_write(dev, 0x1D, val);
+	if (ret)
+		return ret;
+
+	val = (dev->bleed_word >> 8) & ADF4368_BLEED_MSB_MSK;
+	ret = adf4368_spi_update_bits(dev, 0x1E, ADF4368_BLEED_MSB_MSK,
+				      no_os_field_prep(ADF4368_BLEED_MSB_MSK,
+						      val));
+	if (ret)
+		return ret;
+
+	return 0;
+}
+/**
+ * @brief Computes the second fractional part of the feedback divider if needed.
+ * @param dev 	     - The device structure.
+ * @param res 	     - Residue from the first fractional part.
+ * @param pfd_freq   - Phase/frequency detector frequency.
+ * @param frac2_word - Second fractional part of the feedback divider, which
+ * 		       will be returned.
+ * @param mod2_word  - Modulo for the second fractional part of the feedback
+ * 		       divider, which will be returned.
+ * @return 	     - 0 in case of success, negative error code otherwise.
+ */
+static int adf4368_frac2_compute(struct adf4368_dev *dev, uint64_t res,
+				 uint64_t pfd_freq, uint32_t *frac2_word,
+				 uint32_t *mod2_word)
+{
+	uint32_t channel_spacing;
+	uint8_t en_phase_resync;
+	uint32_t chsp_freq;
+	uint32_t mod2_tmp;
+	uint32_t mod2_max;
+	uint32_t mod2_wd;
+	uint32_t gcd;
+	uint8_t tmp;
+	int ret;
+
+	channel_spacing = 1;
+	mod2_wd = 1;
+
+	ret = adf4368_spi_read(dev, 0x1E, &tmp);
+	if (ret)
+		return ret;
+
+	en_phase_resync = no_os_field_get(tmp, ADF4368_EN_PHASE_RESYNC_MSK);;
+
+	if (en_phase_resync)
+		mod2_max = ADF4368_PHASE_RESYNC_MOD2WORD_MAX;
+	else
+		mod2_max = ADF4368_MOD2WORD_MAX;
+
+	do {
+
+		chsp_freq = channel_spacing * ADF4368_MOD1WORD;
+		gcd = no_os_greatest_common_divisor(chsp_freq, pfd_freq);
+		mod2_tmp = NO_OS_DIV_ROUND_UP(pfd_freq, gcd);
+
+		if (mod2_tmp > mod2_max) {
+			channel_spacing *= 5;
+		} else {
+			mod2_wd = mod2_tmp;
+			break;
+		}
+
+	} while (channel_spacing < ADF4368_CHANNEL_SPACING_MAX);
+
+	if (!en_phase_resync) {
```

Comment:
drop brackets.

Reply:
Bracket after "if" is forced with /ci/astyle.sh

Reply:
Fixed